src/helpers/index.js

```diff
@@ -0,0 +1,3 @@
+const regExpEscape = string => string.replace(/[-[\]{}()*+?.,\\^$|]/g, '\\$&');
```

Comment:
Geez... what does it do? If it's something special for searching syntaxes, I'd keep it in the component where we use it.

Reply:
Good place to keep this function to me

Reply:
This mechanism is about escaping regular expression characters

Imagine that we need to create a regular expression for a string. And that string may contain regular expression metacharacters (like $ or .)  that affect the behaviour of the expression. So we need something for escaping these characters.

I think this is a common thing, and you?

Reply:
I think that means we don't need regular expressions then. :) But I'm ok to go your way.

P.S: That's ridiculous that JavaScript doesn't have `escape` function out-of-box.